test/nstemplateset_assertion.go

```diff
@@ -161,6 +161,15 @@ func UpdateFailed(msg string) toolchainv1alpha1.Condition {
 	}
 }
 
+func UnableToProvisionSpaceRoles(msg string) toolchainv1alpha1.Condition {
+	return toolchainv1alpha1.Condition{
+		Type:    toolchainv1alpha1.ConditionReady,
+		Status:  corev1.ConditionFalse,
+		Reason:  toolchainv1alpha1.NSTemplateSetUnableToProvisionSpaceRolesReason,
+		Message: msg,
+	}
+}
+
```

Comment:
where is this used?

Reply:
it's a left-over, not needed. Removed in b4a67cb